src/AureliaPlugin.ts

```diff
@@ -166,7 +166,7 @@ export class AureliaPlugin {
     if (!opts.noHtmlLoader) {
       // Ensure that we trace HTML dependencies (always required because of 3rd party libs)
       let module = compiler.options.module;
-      let rules = module.rules || (module.rules = []);
+      let rules = module.rules || module.loaders || (module.rules = []);
```

Comment:
Isn't it worth logging a warning if `loaders` is used instead of `rules` (and instruct how to update the Webpack configuration)? Or perhaps I have misunderstood smth, as i'm not familiar with Webpack?